packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.tsx

```diff
@@ -45,24 +48,50 @@ export const CombinedRangeFacetFilter: React.FunctionComponent<
   onChange,
   collapsed = false,
 }: CombinedRangeFacetFilterProps) => {
+  const { getCurrentQueryRequest } = useQueryContext()
+
   const [isCollapsed, setIsCollapsed] = useState<boolean>(collapsed)
   const {
-    columnMin: col1Min,
+    columnName: col1Name,
     // columnMax: col1Max  // not used
     selectedMin: col1SelectedMin,
     selectedMax: col1SelectedMax,
   } = facetResults[0]
   const {
+    columnName: col2Name,
     //columnMin: col2Min,  // not used
-    columnMax: col2Max,
     selectedMin: col2SelectedMin,
     selectedMax: col2SelectedMax,
   } = facetResults[1]
 
+  // Also query for the facet columns full range (without any selected facets), to set the range selector range
+  const queryBundleRequest = useMemo(() => {
+    const requestCopy = getCurrentQueryRequest()
+    requestCopy.partMask = SynapseConstants.BUNDLE_MASK_QUERY_FACETS
+    const { selectedFacets } = requestCopy.query
+    requestCopy.query.selectedFacets = selectedFacets
+      ? selectedFacets.filter(
+          facetRequest =>
```

Comment:
Let's keep the other selected facets (so the range could change based on other facet selections, but these two columns should not impact each other!)